contracts/PrepaidCardManager.sol

```diff
@@ -151,6 +159,8 @@ contract PrepaidCardManager is Ownable, Versionable, Safe {
     versionManager = _versionManager;
     Safe.setup(_gsMasterCopy, _gsProxyFactory);
     for (uint256 i = 0; i < _contractSigners.length; i++) {
+      require(_contractSigners[i] != address(0), "contractSigners not set");
```

Comment:
I wanted to suggest `"contractSigner ${i} not set"` but now I see Solidity doesn't support easy string concatenation and it also costs more gas. Do you think it would be clearer to say "contractSigners should not contain empty addresses" or something similar?

Reply:
Yeah I like this wording better, correct that string interpolation in solidity is a pain, and for the sake of a revert message definitely a costly usage of gas